lib/oneflow.js

```diff
@@ -65,8 +69,7 @@ function OneFlowClient(baseUrl, token, secret) {
 		createOrder,
 		orderToJSON,
 		validateOrder,
-		submitOrder,
-		request
```

Comment:
please don't remove this 😢 I use it in so many places

Reply:
Its duplicated, look at the first property of the object.